api/notd/api/endpoints_v1.py

```diff
@@ -163,3 +163,9 @@ class ListCollectionTokenAirdropsRequest(BaseModel):
 
 class ListCollectionTokenAirdropsResponse(BaseModel):
     airdrops: List[ApiAirdrop]
+
+class UpdateAttributeForAllTokensDeferredRequest(BaseModel):
```

Comment:
the plularisation in this is wrong. i think the name can be much better too. it def needs collection in it somwhere. please look at how other messages are named. they follow they same pattern everywhere and it starts with the api path.